samples/drivers/display/sample.yaml

```diff
@@ -79,6 +79,15 @@ tests:
     harness: console
     harness_config:
       fixture: fixture_display
+  sample.display.tn0216anvnann:
+    platform_allow: nrf52dk_nrf52833
```

Comment:
I think you want



```suggestion
    platform_allow: nrf52dk_nrf52832
```

Reply:
thanks for pointing that out! fixed and pushed a commit